src/custom/Stepper/index.tsx

```diff
@@ -58,7 +57,7 @@ const ColorlibConnector = styled(StepConnector)(() => ({
   [`& .${stepConnectorClasses.line}`]: {
     height: 3,
     border: 0,
-    background: '#00B39F',
+    background: theme.palette.grey[700],
```

Comment:
same here

Reply:
This is a token from theme though, is it not?

Reply:
it is theme but not used form palette.
Token means use the colors from palette